src/material/core/style/_focus-indicators.scss

```diff
@@ -0,0 +1,57 @@
+// Private sass variables that will be used as reference throughout component stylesheets.
+$private-default-focus-indicator-border-width: 3px;
+$private-default-focus-indicator-border-style: solid;
+$private-default-focus-indicator-border-color: transparent;
+$private-default-focus-indicator-border-radius: 4px;
+
+// Mixin that renders the focus indicator structural styles.
+@mixin private-structural-styling($prefix) {
```

Comment:
Would it be possible to port over the comments from `_focus-indicators.scss` to this new file? I think some of them are probably helpful to still have.